build/cloud/08-service.yaml.example

```diff
@@ -7,6 +7,7 @@ metadata:
     k8s-app: kubeedge
     kubeedge: cloudcore
 spec:
+  type: NodePort
```

Comment:
why did this change here?

Reply:
I think we can use `hostNetwork` in deployment

Reply:
This was only a mistake, I did this change for my local setup.